config/locales/views/campus/en.yml

```diff
@@ -1,10 +1,16 @@
 en:
+  name: "Name"
+  created_at: "Created at"
+  updated_at: "Updated at"
   save_success: "Campus saved!"
   save_error: "Something went wrong. Could not save campus"
   update_success: "Campus updated"
   update_error: "Something went wrong. Could not update campus"
   number_of_applicants: "Number of applicants"
   number_of_current_applicants: "Number of applicants this admission"
+  campus_edit: "Edit"
   activate: "Activate"
   deactivate: "Deactivate"
+  delete: "Delete"
+  areYouSure: "Are you sure?"
```

Comment:
A better naming convention is "are_you_sure"

Reply:
Perhaps message should be a bit more informative, such as "Are you sure you want to delete this campus?"